cdc/api/v1/api.go

```diff
@@ -695,6 +696,68 @@ func (h *OpenAPI) ListCapture(c *gin.Context) {
 	c.IndentedJSON(http.StatusOK, captures)
 }
 
+// DrainCapture remove all tables at the given capture.
+// @Summary Drain captures
+// @Description Drain all tables at the target captures in cdc cluster
+// @Tags capture
+// @Accept json
+// @Produce json
+// @Success 200,202
+// @Failure 500,400 {object} model.HTTPError
+// @Router	/api/v1/captures/drain [post]
+func (h *OpenAPI) DrainCapture(c *gin.Context) {
+	if !h.capture.IsOwner() {
```

Comment:
We use middleware to forward the traffic to the owner and will the path reachable?